mgmtfn/dockplugin/netDriver.go

```diff
@@ -240,12 +241,24 @@ func createEndpoint(hostname string) func(http.ResponseWriter, *http.Request) {
 
 		log.Debug(ep)
 
+		// need to get the subnetlen from nw state.
+		nw, err := netdGetNetwork(netID)
+		if err != nil {
+			httpError(w, "Could not find created network", err)
+			return
+		}
+
 		epResponse := api.CreateEndpointResponse{
 			Interface: &api.EndpointInterface{
 				MacAddress: mresp.EndpointConfig.MacAddress,
 			},
 		}
 
+		// setup IPv6 address
+		if ep.IPv6Address != "" {
+			epResponse.Interface.AddressIPv6 = ep.IPv6Address + "/" + strconv.Itoa(int(nw.IPv6SubnetLen))
```

Comment:
why do you need to set ipv6 address in response ? its already there in the request